assets/js/state/lastExecutions.js

```diff
@@ -42,11 +44,36 @@ export const lastExecutionsSlice = createSlice({
 
       const lastExecutionState = {
         ...initialExecutionState,
+        executionStarted: true,
         error,
       };
 
       state[groupID] = lastExecutionState;
     },
+    setExecutionStarted: (state, { payload }) => {
+      const { groupID: clusterID, checks: startedChecks } = payload;
+
+      // The execution started event contains the information
+      // about the checks "really" started in the execution
+      // this checks could differ from the ones requested
+      // because for some reason the checks could be skipped by
+      // some conditions
+      // we want to make sure that only the real checks executed
+      // are present into the state when an execution occur
+
+      // eslint-disable-next-line prefer-destructuring
+      const targets = state[clusterID].data.targets;
```

Comment:
I think this is totally doable by the frontend, I discussed with @fabriziosestito and this + the web transformation seems like a good solution

Reply:
if the targets are == to what we have in the event, the transformation is superfluous. I suggested just having the checks in the broadcasted event since I wrongly believed that we needed just the list of the checks selection. I now realize that the FE is working with a targets-based hierarchy so I agree with @arbulu89 on this one.

Reply:
done